include/mcptam/EntropyComputation.h

```diff
@@ -0,0 +1,41 @@
+/****************************************************************************************
+ *
+ * \file EntropyComputation.h
+ * \brief Functions for entropy computations
+ *
+ * Copyright 2015   Arun Das, University of Waterloo (adas@uwaterloo.ca)
+ *
+ *
```

Comment:
Added MCPTAM copyright header and the following discription:

 * This class contains functions for calculating the entropy (uncertainty) of state
 * parameters in tracker to decide when a keyframe should be selected. Once the individual 
 * entropy of a state rises above a user-defined threshold, a multi-keyframe that maximizes
 * the reduction in expected entropy is found and inserted in the map.
 *
 * For more details visit: A. Das and S. L. Waslander, “Entropy based keyframe selection for
 * multi-camera visual slam,” in Ieee/rsj international conference on intelligent robots and
 * systems (iros), Hamburg, Germany, 2015, pp. 3676-3681.